### PR TITLE
[FIX] odoo_sh: change img who explain storage space

### PR DESCRIPTION
**Pull request by roussieau**
The image that explains the storage in odoo.sh is outdated. 

https://www.odoo.com/web#id=2948233&cids=1&menu_id=4720&action=333&active_id=1105&model=project.task&view_type=form

### CODE REVIEW

**Comment by amigrave** [comment]
@robodoo r+

**Review by Lau-Odoo — APPROVED** [review]

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
Hello @roussieau 

Thanks for your contribution to the doc.

- Image files must be compressed with pngquant (see [content guidelines: images](https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#images))
- you target 15.0 on this PR which is probably a mistake, considering the page is the same from 13.0. To change the target of the PR, please set it as a draft before you do this change, otherwise, all codeowners of this repo would get a notification.

Thanks :)

@amigrave you can't r+ on this repo I'm afraid. Nothing to do, though. Once a PR is ready for review, the right codeowners get notified.

@Lau-Odoo you must check if images are compressed and make sure the target is right before you approve a PR.

**Comment by roussieau** [comment]
I opened a new PR that targets the 13.0 with the compressed image. #2607

**Pull request closed without merging**
